sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
-
+- Various bug fixes in Administration client. ([#31712](https://github.com/Azure/azure-sdk-for-java/issues/31712))
```

Comment:
`Various bug fixes` seem not straightforward to customer, although they can click to check this PR. 
I not sure if it is good to mention change content, things like fix `createQueue()`, `createWithResponse()` and all list methods failed to be called in administration client?

Reply:
Ah I thought listing all the methods would be very verbose. which is why I thought just having the link to the issue would give them more details there. I am okay with either of the options though.

Reply:
I think listing them is better. When I see if a fix applies to me, following another level of indirection is cumbersome.

Reply:
updated